iam/x509-identity-mgmt/js/schemas/register-trusted-ca-certificate.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "http://www.dojot.com.br/schemas/reg-trust-ca",
+  "$id": "https://raw.githubusercontent.com/dojot/dojot/development/iam/x509-identity-mgmt/js/schemas/register-trusted-ca-certificate.json",
```

Comment:
I like the change

Reply:
It would be nice to have a treatment to define the correct tag in the URL, but we'll leave that for the next one.